packages/launch/src/stores/index.ts

```diff
@@ -0,0 +1,12 @@
+// this file is a copy of apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/stores/index.ts
+// TODO: avoid code duplication
+
+/**
+ * External dependencies
+ */
+import 'a8c-fse-common-data-stores';
+
+export const SITE_STORE = 'automattic/site';
+export const LAUNCH_STORE = 'automattic/launch';
+export const PLANS_STORE = 'automattic/onboard/plans';
+export const DOMAINS_STORE = 'automattic/domains/suggestions';
```

Comment:
Data stores are now registered directly in this PR. https://github.com/Automattic/wp-calypso/pull/46834